src/Manager.cpp

```diff
@@ -450,4 +450,10 @@ Manager::listDevices() const
     return this->mInstance->enumeratePhysicalDevices();
 }
 
+std::shared_ptr<vk::Instance>
+Manager::getVkInstance() const
+{
+    return mInstance;
```

Comment:
The current coneebase standard is to use explicit vs implicit with `this->` for member attributes

Reply:
Ah, right. Fixed.